app.js

```diff
@@ -22,4 +23,29 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then(beersFromApi => {     
+      res.render('beers',{beersFromApi});
+    })
+    .catch(error => console.log(error))
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI.getRandom()
+    .then(responseFromApi => {     
+      res.render('randomBeer',{responseFromApi});
+    })
+    .catch(error => console.log(error))
+});
```

Comment:
It is your preference, but my recommendation would be to format data in javascript, before you even send info to the view -> this way your view files will be short and easy to read.

So for example, on line 37, instead of sending an array with one element, you can send that single object directly

```
    .then(responseFromApi => {     
      res.render('randomBeer', responseFromApi[0]);
    })
```